app/models/miq_report_result.rb

```diff
@@ -11,10 +11,8 @@ class MiqReportResult < ApplicationRecord
   serialize :report
 
   virtual_delegate :description, :to => :miq_group, :prefix => true, :allow_nil => true
-  virtual_delegate :state_or_status, :to => "miq_task", :allow_nil => true
-  virtual_attribute :status, :string, :uses => :state_or_status
```

Comment:
Search for string `state_or_status`across all projects: found only in `MiqTask`  and it is not invocation of `MiqTaskResult.state_or_status`